components/spi/config-patch.json

```diff
@@ -2,11 +2,11 @@
   {
     "op": "replace",
     "path": "/data/VAULTHOST",
-    "value": "https://spi-vault-spi-vault.apps.appstudio-kcp.g2pi.p1.openshiftapps.com"
+    "value": "https://vault-spi-vault.apps.appstudio-kcp.g2pi.p1.openshiftapps.com"
   },
   {
     "op": "replace",
     "path": "/data/BASEURL",
-    "value": "https://spi-oauth-kcp-2l71pp1tc2f1.apps.appstudio-kcp.g2pi.p1.openshiftapps.com"
+    "value": "https://spi-oauth-kcp-5ttoh8uv3y7i.apps.appstudio-kcp.g2pi.p1.openshiftapps.com"
```

Comment:
but this would mean that it would use one endpoint for both kcp instances, is this what you want?